From e5d7c2cfb98e2c2442c147ac2c76d00e6512d69c Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Michael=20M=C3=BCller?= <michael@fds-team.de>
Date: Sun, 13 Mar 2016 03:59:22 +0100
Subject: wine.inf.in: Add invalid dummy certificate to CA certificate store.

Pythons ssl module raises an exception if the CA certificate store is empty.
Wine currently doesn't add any certificates to this store and this patch tries
to solve the problem by adding a dummy certificate. The certificate shouldn't
have any effect though as it is invalid by design. The dates were choosen in a
way that this certificate can never be valid:

Not Before: Mar 13 02:13:31 2016 GMT
Not After : Mar 12 02:13:31 2016 GMT
---
 loader/wine.inf.in | 43 +++++++++++++++++++++++++++++++++++++++++++
 1 file changed, 43 insertions(+)

diff --git a/loader/wine.inf.in b/loader/wine.inf.in
index d22b29b..6ee74a5 100644
--- a/loader/wine.inf.in
+++ b/loader/wine.inf.in
@@ -652,6 +652,49 @@ HKLM,Software\Microsoft\Rpc\SecurityService,68,2,"netlogon.dll"
 HKLM,Software\Microsoft\Rpc\SecurityService,9,2,"secur32.dll"
 HKLM,Software\Microsoft\Shared Tools\MSInfo,,16
 HKLM,Software\Microsoft\SystemCertificates,,16
+HKLM,Software\Microsoft\SystemCertificates\CA\Certificates\23340A0167398E341C5890709FD30EED44FC1964,"Blob",1,\
+    03,00,00,00,01,00,00,00,14,00,00,00,23,34,0a,01,67,39,8e,34,1c,58,90,70,9f,d3,0e,ed,44,fc,19,64,20,00,00,\
+    00,01,00,00,00,81,05,00,00,30,82,05,7d,30,82,03,65,a0,03,02,01,02,02,09,00,87,0d,26,f3,cb,4a,11,1b,30,0d,\
+    06,09,2a,86,48,86,f7,0d,01,01,0d,05,00,30,55,31,0b,30,09,06,03,55,04,06,13,02,55,53,31,13,30,11,06,03,55,\
+    04,08,0c,0a,53,6f,6d,65,2d,53,74,61,74,65,31,0d,30,0b,06,03,55,04,0a,0c,04,57,69,6e,65,31,22,30,20,06,03,\
+    55,04,03,0c,19,49,6e,76,61,6c,69,64,20,64,75,6d,6d,79,20,63,65,72,74,69,66,69,63,61,74,65,30,1e,17,0d,31,\
+    36,30,33,31,33,30,32,31,33,33,31,5a,17,0d,31,36,30,33,31,32,30,32,31,33,33,31,5a,30,55,31,0b,30,09,06,03,\
+    55,04,06,13,02,55,53,31,13,30,11,06,03,55,04,08,0c,0a,53,6f,6d,65,2d,53,74,61,74,65,31,0d,30,0b,06,03,55,\
+    04,0a,0c,04,57,69,6e,65,31,22,30,20,06,03,55,04,03,0c,19,49,6e,76,61,6c,69,64,20,64,75,6d,6d,79,20,63,65,\
+    72,74,69,66,69,63,61,74,65,30,82,02,22,30,0d,06,09,2a,86,48,86,f7,0d,01,01,01,05,00,03,82,02,0f,00,30,82,\
+    02,0a,02,82,02,01,00,ec,26,db,20,a6,25,fc,6d,d7,7f,35,45,2a,bd,26,9a,6c,a7,f3,4c,e7,d4,1b,c1,44,bf,13,61,\
+    9c,93,9f,ff,f2,89,d7,aa,52,d0,e0,d3,41,99,00,a5,82,0e,71,e2,f7,22,0c,9c,3a,21,e1,b2,ba,a8,e2,0b,6f,7c,7c,\
+    fb,92,c4,98,9f,01,dd,17,b4,f7,ac,ab,75,40,fb,b8,1d,7c,4f,c2,ac,6f,d5,ed,38,b6,b5,dd,73,fe,3d,21,f7,fd,81,\
+    e6,af,fe,85,a4,0b,9c,d2,d2,7f,fd,45,3e,7f,80,ce,69,69,38,e1,5c,9c,bc,fd,4c,8b,64,78,f7,45,56,53,8f,97,aa,\
+    98,3a,49,37,2e,45,a6,dc,04,fc,89,26,71,fd,15,24,3b,66,5c,3d,fe,7f,ef,25,b3,01,94,a1,8f,83,5d,b0,d7,d5,1c,\
+    8e,14,b2,95,f5,52,28,50,ab,81,74,0e,0f,5f,61,fe,4b,03,e0,c4,98,3a,f9,01,c7,f0,c9,66,12,d9,60,17,a9,1e,6b,\
+    2d,91,6b,de,95,96,58,f5,81,21,af,d1,7f,99,01,cb,e6,1d,cf,d3,0f,1e,70,4b,de,2b,2c,fe,1f,1d,d8,76,32,74,8d,\
+    bc,4f,0c,33,6d,3b,9f,ba,15,16,28,70,44,6b,da,c5,85,11,4f,bd,4c,00,b4,f1,03,93,89,18,63,1d,ec,3e,ad,6a,b1,\
+    fc,07,f7,65,7c,bf,6a,c8,e7,2a,19,40,0a,55,59,39,63,b1,b8,a8,95,20,62,ae,97,82,af,d6,b1,97,ce,ba,29,19,9f,\
+    dd,c8,63,a0,80,34,e5,a0,c5,f6,b2,95,b1,8e,34,39,34,08,cb,24,70,a7,fb,8e,34,2c,e4,41,77,ea,05,3b,30,5e,71,\
+    b9,64,9f,bf,a6,db,d8,ef,5e,42,9c,4e,5d,47,68,a5,23,15,4f,07,3c,d3,ea,62,ff,af,4c,08,e7,9b,2b,9c,0a,69,f7,\
+    3e,ab,8a,05,40,d2,17,7e,0f,c8,37,2b,e2,62,25,f7,e9,e0,d7,8f,18,ff,46,f9,35,3f,2d,d6,3e,cf,e2,fd,08,0d,1f,\
+    63,ff,80,75,50,3b,a9,44,11,c8,b4,ae,a4,24,17,e3,0d,8a,a5,d3,56,37,97,99,91,c1,62,f9,75,a6,77,e0,25,f6,a4,\
+    9c,83,36,bb,4a,c5,82,c4,86,03,2f,dd,6f,c4,45,ec,fe,d0,51,d6,e0,d2,6a,da,00,30,24,c0,8e,e8,a1,ce,4c,7f,bb,\
+    54,09,05,9e,2b,93,95,c8,ac,39,12,29,56,4e,b0,20,36,9c,a3,b6,cb,46,59,a3,2b,62,2c,2d,57,02,03,01,00,01,a3,\
+    50,30,4e,30,1d,06,03,55,1d,0e,04,16,04,14,d1,85,39,b7,9b,05,e7,f9,54,12,91,68,ed,93,ec,63,eb,d3,8f,14,30,\
+    1f,06,03,55,1d,23,04,18,30,16,80,14,d1,85,39,b7,9b,05,e7,f9,54,12,91,68,ed,93,ec,63,eb,d3,8f,14,30,0c,06,\
+    03,55,1d,13,04,05,30,03,01,01,ff,30,0d,06,09,2a,86,48,86,f7,0d,01,01,0d,05,00,03,82,02,01,00,be,a7,cf,6e,\
+    e7,3d,5d,f2,38,0c,f9,69,85,fe,79,25,cf,c5,5b,d3,4e,4b,ec,20,16,1b,46,1a,d9,a4,a1,84,24,9a,4d,0d,44,70,83,\
+    be,e3,af,5b,ac,c4,31,e3,d5,f8,cc,af,71,11,2d,ea,a8,44,20,81,8b,e3,f6,da,77,6a,f6,15,f3,11,78,78,76,4f,6d,\
+    b6,8c,a5,27,41,0a,16,b6,c8,56,dc,a6,a2,0a,e2,87,e9,ab,f0,05,8b,61,7c,a7,03,2f,30,cc,dd,21,7e,48,43,10,cd,\
+    aa,9e,37,69,7c,3e,2f,7b,c1,0c,78,80,aa,18,92,95,a0,2a,9e,2f,a6,33,6b,91,8d,fb,f1,47,16,ec,87,86,7a,26,d3,\
+    57,a9,89,37,76,86,76,9b,96,bf,f1,2a,e3,62,2a,bc,66,7b,91,62,fb,a2,7e,16,c6,6e,84,08,49,b0,9d,5d,42,b7,64,\
+    39,1f,49,a6,a2,98,bc,ca,57,e5,f5,91,c0,6b,6c,94,56,68,34,00,96,25,1d,e5,bb,9e,2f,1a,dd,c2,27,e4,49,f2,54,\
+    68,0a,e3,1b,4c,01,14,0d,64,8c,d6,ab,73,af,51,b6,fe,af,0a,b6,c1,99,36,97,f8,44,48,64,d3,c0,ab,d9,a8,da,06,\
+    1a,27,c3,24,cf,7a,30,12,38,20,6c,71,ea,57,fa,b6,24,23,42,3d,be,07,f9,d7,de,b2,91,20,a6,57,79,06,60,42,cb,\
+    73,2b,98,ea,49,78,ee,47,71,a5,1c,f1,f9,8f,78,49,ad,6f,21,72,1a,d2,90,02,59,01,7d,f1,cd,c7,43,4c,f4,6c,90,\
+    59,38,bf,46,92,09,24,3b,51,2b,49,2a,a7,f9,7c,96,44,e6,ed,8e,7d,85,64,6b,ca,7b,e3,af,bd,73,ae,17,31,52,2a,\
+    9f,a3,3b,dd,3d,33,59,26,7c,6d,46,ff,5b,dd,3c,82,dd,d1,3d,9c,31,2a,82,7e,85,ce,ef,a6,34,98,96,87,7a,2b,a0,\
+    a5,5a,9c,2a,5c,f4,79,36,b4,85,87,eb,fd,7e,94,83,40,b0,4f,d7,03,c1,df,75,37,30,4f,46,6b,29,b5,0f,b7,4f,19,\
+    73,79,dd,ee,2d,2c,c8,1f,72,ee,63,4a,32,85,33,57,1d,f5,89,53,4f,b2,1e,20,9b,11,e4,43,99,15,65,c4,a0,ec,cf,\
+    d3,0f,d6,b2,2c,8c,a7,6d,5f,11,2f,0e,e4,f7,e9,d4,13,db,fb,f6,0a,51,ac,6c,08,ef,d6,cb,81,86,53,36,a8,a0,02,\
+    de,31,47,56,db,65,f1,7e,3a,c8,69,5b,69,ab,34,3e,48,a3
 HKLM,Software\Microsoft\TermServLicensing,,16
 HKLM,Software\Microsoft\Transaction Server,,16
 HKLM,Software\Microsoft\WBEM,"Installation Directory",2,"%11%\wbem"
-- 
2.7.1

